release-team/role-handbooks/patch-release-manager/README.md

```diff
@@ -380,3 +391,26 @@ or push tags or release artifacts to public repositories until the PST tells you
 See the [Security Release Process](https://git.k8s.io/security/security-release-process.md)
 doc for more details.
 
+## Release Commands Cheat Sheet
+
+| Action | Example flow for 1.13.3 |
+| --- |--- |
+| Make sure you have latest release tooling | ```cd ~/go/src/k8s.io/release && git pull``` |
+| Configure branch | n/a |
+| Mock build staging | ```./gcbmgr stage release-1.13 ``` This may fail (eg: CI not fully green), and if so gives hint on suggested buildversion for explicit use. |
```

Comment:
How about adding the code formatting for: _...suggested `buildversion` for explicit use._

Reply:
Also, perhaps have _... and if so, it gives hint on the sugessted ..._ instead? If you're trying to use less words, then I'd say ignore this.

Reply:
There's two rows:
* "Mock build staging"
* "Mock build staging, optional..."

I don't know how to make that fork more clear visually in a table.

I think it's reasonable to always try to run without buildversion and then fallback next to specifying it.

I tried making it more clear in a revision of the commit.  What do you think?

Reply:
Like what you've done there. :smile: thanks!